src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2783,7 +2987,8 @@ because xbuild doesn't support framework reference assemblies.
 	<Delete Files="$(MonoAndroidIntermediate)__AndroidLibraryProjects__.zip" />
 	<Delete Files="$(MonoAndroidIntermediate)__AndroidNativeLibraries__.zip" />
 	<Delete Files="$(MonoAndroidIntermediate)stub_application_data.txt" />
-	<Delete Files="$(IntermediateOutputPath)_javac.stamp" />
+	<Delete Files="$(MonoAndroidIntermediate)_javac.stamp" />
```

Comment:
Is this correct? The only other appearance of `_javac.stamp` I see is in `_CompileJava`:

```
<Target Name="_CompileJava"
  DependsOnTargets="$(_CompileJavaDependsOnTargets)"
  Inputs="$(MSBuildAllProjects);$(MonoPlatformJarPath);@(_JavaStubFiles);@(AndroidJavaSource);@(AddOnPlatformLibraries)"
  Outputs="$(IntermediateOutputPath)_javac.stamp">
  ...
  <Touch Files="$(IntermediateOutputPath)_javac.stamp" AlwaysCreate="true" />
```

Thus, `$(IntermediateOutputPath)` looks correct.